### ISSUE TITLE
Fix WCAG 2.4.4: Link Purpose issues

### ISSUE THREAD

**Issue by juliatufts** [basekamp/basekamp-next#9]
Fix the following issues related to WCAG 2.4.4: Link Purpose, see also: [full accessibility report](https://github.com/basekamp/basekamp-next/wiki/Accessibility-Evaluation-Report-2026).

## About

Suggest linkifying "join us" instead of "Let us know"

## Projects

Links are repeated, should just name as project title, and group together with image

## Events

Links are repeated, should just name as event title, and group together with image

## Event page (Homework)

Unsure if raw http link is failure, probably better to name the link.
